app/google-services.json

```diff
@@ -13,6 +13,14 @@
         }
       },
       "oauth_client": [
```

Comment:
this is service file for firebase, it generated from firebase and should be updated in case of changes in some changes in firebase, that was updated when I tried reset firebase and fb auth